docs/upgrade.html

```diff
@@ -19,7 +19,7 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
-<h4><a id="upgrade_2_6_1" href="#upgrade_2_6_1">Upgrading to 2.6.1 from any version 0.8.x through 2.5.x</a></h4>
+<h4><a id="upgrade_2_6_2" href="#upgrade_2_6_2">Upgrading to 2.6.2 from any version 0.8.x through 2.5.x</a></h4>
```

Comment:
Since the 2.6.2 release is already under way, should we just bump this straight to 2.6.3 so we're prepared for the next bugfix release (if we ever get one)?

Reply:
The docs have already been generated for 2.6.2, so this won't get picked up unless we have to roll a new RC for some reason. How about we bump it to 2.6.3, but I'll wait until the current RC has been approved until merging this just in case -- thoughts?

Reply:
Thanks, I just thought we missed it. If not, I'll close this PR, then. 
Thank you for the V2.6.2 release. :)

Reply:
No worries! And actually I wasn't implying that we should close this PR, just that I'll need to do a separate one against kafka-site since the docs have already been generated for 2.6.2. If you reopen this PR I can go ahead and merge it 

Once again, thanks for catching this! It looks like this has been missed before (eg the 2.5 branch still references 2.4.0 when it should be up to 2.5.1) -- guess it's easy to miss. I'll add a note to the release guide to point this out explicitly for future RMs

Reply:
Noticed that the 2.8 branch was also missing this so I brought this to the 2.8 RM's attention too,  thanks again for the attention to detail!